Header header
uint16 reg_motor_fault_flag_left
uint16 reg_motor_temp_left
uint16 reg_motor_temp_right
uint16 reg_power_bat_voltage_a
uint16 reg_power_bat_voltage_b
uint16 reg_robot_rel_soc_a
uint16 reg_robot_rel_soc_b
uint16 battery_mode_a
uint16 battery_mode_b
uint16 battery_temp_a
uint16 battery_temp_b
uint16 battery_voltage_a
uint16 battery_voltage_b
uint16 buildno
